content/doc/developer/plugin-development/build-process.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: developersection
+title: Plugin Build Process
+references:
+- url: ../updating-parent/
+  title: Guide to Updating Your Maven Parent POM
+- url: https://github.com/jenkinsci/plugin-pom/
+  title: Plugin parent POM 2.x and up
+- url: https://github.com/jenkinsci/maven-hpi-plugin/
+  title: Maven HPI Plugin
+---
+
+Most Jenkins plugins are built with link:https://maven.apache.org[Maven]. The link:https://github.com/jenkinsci/maven-hpi-plugin/[Maven HPI plugin] does the heavy lifting, such as bundling plugins in the HPI/JPI archive format, or running a debug Jenkins instance with the plugin.
+
+Jenkins plugins typically use the Jenkins plugin parent POM, which provides a sensible default configuration for the build.
```

Comment:
"use the Jenkins plugin POM as their parent"?

Linking to the plugin-pom repo here would be nice (even if it is in references).

Reply:
Briefly mentioning what a POM is, or at least providing a link (e.g. https://maven.apache.org/pom.html), would be helpful.

Reply:
It seems like these paragraphs should perhaps be in a different order, e.g.:
Jenkins plugins are built with Maven. They use the Jenkins plugin POM as their parent. This provides various tools, including the Maven HPI Plugin, which does all the magic.

Reply:
Similarly, since this is the page about how the plugin build process, it seems like there should be an explanation of what a "HPI/JPI" is, and potentially why we have a special plugin for it…